core/libcamera_app.hpp

```diff
@@ -194,6 +195,7 @@ class LibcameraApp
 	Stream *still_stream_ = nullptr;
 	Stream *raw_stream_ = nullptr;
 	Stream *video_stream_ = nullptr;
+	Stream *lores_stream_ = nullptr;
 	FrameBufferAllocator *allocator_ = nullptr;
```

Comment:
I wonder if we should replace this a std::map<std::string, Stream *> to make things a bit easier?
Then all our accessor functions can also disappear.

Reply:
Yes, I agree that maybe something like a map might be tidier now. Are you thinking of something like "app->Stream("still")" in place of "app->StillStream()", or of exposing the map itself? I think I still prefer "if (app->StillStream()) ..." slightly to "if (app->Streams().find("still") != app->Streams().end()) ..."

Reply:
You could use if (app->Streams("still").count) as a shortcut, but I am not too fussed wither way.